integrationTests/basic/vitest.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    // required to make jest-when work with vitest
+    globals: true,
+    exclude:[
+      ...configDefaults.exclude,
+      './src/**',
+      '**/integrationTests/e2e/**',
+      '**/integrationTests/e2e-tls/**',
+      '**/integrationTests/enterprise/**',
```

Comment:
I wonder if this we can replace excluding these with https://vitest.dev/config/#include in `integrationTests/<test-module>/vitest.config.js` files.
```
exclude:[
      ...configDefaults.exclude,
     './src/**',
     '**/integrationTests/**',
],
include: [
     '**/integrationTests/basic/**',
]
```
I think it'll require a little more effort to maintain if we have new test modules under `integrationTests`.

Sorry I might be missing the difference between the `exclude` and `include` approaches here as I'm picking up `vitest`. Let me know your thought.

Reply:
Yeah, I agree this isn't ideal but it was the best I could come up with. When I use your suggestion I get `No test files found, exiting with code 1`. I think the exclude overrides the include unfortunately.

Reply:
ok I think I came up with a solution in https://github.com/hashicorp/vault-action/pull/533/commits/848a30c5aee2137e8e32c22c8186c4bb634b73b5. Thanks for the suggestion @thy!

Reply:
The pattern matching is cool!

Reply:
My username @thyton by the way. I hope the other use is pinged as this is a private repo.